### ISSUE TITLE
test failure for qa_correlate_access_code_XX_ts.py

### ISSUE THREAD

**Issue by ZeroChaos-** [gnuradio/gnuradio#7566]
### What happened?

```
181/267 Test #192: qa_correlate_access_code_XX_ts ...............***Failed    0.34 sec
......E
======================================================================
ERROR: test_payload_lengths (__main__.test_correlate_access_code_XX_ts.test_payload_lengths)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/var/tmp/portage/net-wireless/gnuradio-9999/work/gnuradio-9999/gr-digital/python/digital/qa_correlate_access_code_XX_ts.py", line 104, in test_payload_lengths
    self._variable_payload_t(length)
  File "/var/tmp/portage/net-wireless/gnuradio-9999/work/gnuradio-9999/gr-digital/python/digital/qa_correlate_access_code_XX_ts.py", line 77, in _variable_payload_t
    self.assertEqual(header[0] * 256 + header[1], header[2] * 256 + header[3])
                     ~~~~~~~~~~^~~~~
OverflowError: Python integer 256 out of bounds for uint8
```

### System Information

OS:  Gentoo
GR Installation Method: Native distro package (compiling with package manager help)

### GNU Radio Version

3.10 (maint-3.10)

### Specific Version

v3.10.11.0-1-gee27d6f3

### Steps to Reproduce the Problem

Build and run tests

### Relevant log output

```shell
>>> Emerging (1 of 1) net-wireless/gnuradio-9999::maintainer-zero
 * Using python3.12 to build
>>> Unpacking source...
 * Repository id: gnuradio_gnuradio.git
 * To override fetched repository properties, use:
 *   EGIT_OVERRIDE_REPO_GNURADIO_GNURADIO
 *   EGIT_OVERRIDE_BRANCH_GNURADIO_GNURADIO
 *   EGIT_OVERRIDE_COMMIT_GNURADIO_GNURADIO
 *   EGIT_OVERRIDE_COMMIT_DATE_GNURADIO_GNURADIO
 * 
 * Fetching https://github.com/gnuradio/gnuradio.git ...
git fetch https://github.com/gnuradio/gnuradio.git +refs/heads/maint-3.10:refs/heads/maint-3.10
git symbolic-ref refs/git-r3/net-wireless/gnuradio/0/__main__ refs/heads/maint-3.10
 * Checking out https://github.com/gnuradio/gnuradio.git to /var/tmp/portage/net-wireless/gnuradio-9999/work/gnuradio-9999 ...
git checkout --quiet maint-3.10
GIT update -->
   repository:               https://github.com/gnuradio/gnuradio.git
   at the commit:            ee27d6f3d63b39e1427f80de9950d9a64bdae096
>>> Source unpacked in /var/tmp/portage/net-wireless/gnuradio-9999/work
>>> Preparing source in /var/tmp/portage/net-wireless/gnuradio-9999/work/gnuradio-9999 ...
 * Source directory (CMAKE_USE_DIR): "/var/tmp/portage/net-wireless/gnuradio-9999/work/gnuradio-9999"
 * Build directory  (BUILD_DIR):     "/var/tmp/portage/net-wireless/gnuradio-9999/work/gnuradio-9999_build"
>>> Source prepared.
>>> Configuring source in /var/tmp/portage/net-wireless/gnuradio-9999/work/gnuradio-9999 ...
 * Source directory (CMAKE_USE_DIR): "/var/tmp/portage/net-wireless/gnuradio-9999/work/gnuradio-9999"
 * Build directory  (BUILD_DIR):     "/var/tmp/portage/net-wireless/gnuradio-9999/work/gnuradio-9999_build"
cmake -C /var/tmp/portage/net-wireless/gnuradio-9999/work/gnuradio-9999_build/gentoo_common_config.cmake -G Ninja -DCMAKE_INSTALL_PREFIX=/usr -DENABLE_DEFAULT=OFF -DENABLE_GNURADIO_RUNTIME=ON -DENABLE_PYTHON=ON -DENABLE_GR_BLOCKS=ON -DENABLE_GR_ANALOG=ON -DENABLE_GR_AUDIO=ON -DENABLE_GR_CHANNELS=ON -DENABLE_GR_CTRLPORT=OFF -DENABLE_GR_DIGITAL=ON -DENABLE_DOXYGEN=OFF -DENABLE_GR_DTV=ON -DENABLE_GR_FEC=ON -DENABLE_GR_FFT=ON -DENABLE_GR_FILTER=ON -DENABLE_GRC=ON -DENABLE_GR_IIO=ON -DENABLE_GR_MODTOOL=ON -DENABLE_GR_BLOCKTOOL=ON -DENABLE_GR_NETWORK=ON -DENABLE_GR_PDU=ON -DENABLE_PERFORMANCE_COUNTERS=OFF -DENABLE_TESTING=ON -DENABLE_GR_QTGUI=ON -DENABLE_GR_SOAPY=ON -DENABLE_GR_TRELLIS=ON -DENABLE_GR_UHD=ON -DENABLE_GR_UTILS=ON -DENABLE_GR_VIDEO_SDL=ON -DENABLE_GR_VOCODER=ON -DENABLE_GR_WAVELET=ON -DENABLE_GR_ZEROMQ=ON -DSYSCONFDIR=/etc -DPYTHON_EXECUTABLE=/usr/bin/python3.12 -DGR_PYTHON_DIR=/usr/lib/python3.12/site-packages -DGR_PKG_DOC_DIR=/usr/share/doc/gnuradio-9999 -DMATHJAX2_ROOT=/usr/share/mathjax -DCMAKE_BUILD_TYPE=None -DCMAKE_TOOLCHAIN_FILE=/var/tmp/portage/net-wireless/gnuradio-9999/work/gnuradio-9999_build/gentoo_toolchain.cmake /var/tmp/portage/net-wireless/gnuradio-9999/work/gnuradio-9999
loading initial cache file /var/tmp/portage/net-wireless/gnuradio-9999/work/gnuradio-9999_build/gentoo_common_config.cmake
-- The CXX compiler identification is GNU 14.2.1
-- The C compiler identification is GNU 14.2.1
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/x86_64-pc-linux-gnu-g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/x86_64-pc-linux-gnu-gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Build type set to None.
-- Build date set to Tue, 17 Dec 2024 21:00:04Z.
-- Found Git: /usr/bin/git
-- Extracting version information from git describe...
CMake Warning (dev) at cmake/Modules/GrBoost.cmake:111 (find_package):
  Policy CMP0167 is not set: The FindBoost module is removed.  Run "cmake
  --help-policy CMP0167" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

Call Stack (most recent call first):
  CMakeLists.txt:62 (include)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found Boost: /usr/lib64/cmake/Boost-1.84.0/BoostConfig.cmake (found suitable version "1.84.0", minimum required is "1.69") found components: date_time program_options system regex thread unit_test_framework
-- 
-- Configuring testing-support support...
--   Dependency Boost_UNIT_TEST_FRAMEWORK_FOUND = 1
--   Enabling testing-support support.
--   Override with -DENABLE_TESTING=ON/OFF
-- Performing Test HAVE_VISIBILITY_HIDDEN
-- Performing Test HAVE_VISIBILITY_HIDDEN - Success
-- Performing Test HAVE_WARN_SIGN_COMPARE
-- Performing Test HAVE_WARN_SIGN_COMPARE - Success
-- Performing Test HAVE_WARN_ALL
-- Performing Test HAVE_WARN_ALL - Success
-- Performing Test HAVE_WARN_NO_UNINITIALIZED
-- Performing Test HAVE_WARN_NO_UNINITIALIZED - Success
-- Performing Test HAVE_WARN_IGNORED_QUALIFIERS
-- Performing Test HAVE_WARN_IGNORED_QUALIFIERS - Success
-- Performing Test HAVE_WARN_CAST_QUAL
-- Performing Test HAVE_WARN_CAST_QUAL - Success
-- Compiler Version: x86_64-pc-linux-gnu-gcc (Gentoo Hardened 14.2.1_p20241116 p3) 14.2.1 20241116
Copyright (C) 2024 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
-- Compiler Flags: /usr/bin/x86_64-pc-linux-gnu-gcc::: -mtune=core2 -pipe -O3 -flto -frecord-gcc-switches -Wstringop-overread -Wformat -Wformat-security -D_FORTIFY_SOURCE=3 -D_GLIBCXX_ASSERTIONS -ftrivial-auto-var-init=pattern -Wl,-z,relro,-z,now -fstack-protector-strong -fstack-clash-protection -fcf-protection=full -Werror=strict-aliasing -Werror=odr -Werror=lto-type-mismatch -Werror=stringop-overread -Werror=format-security -march=alderlake -mabm -mno-cldemote -mno-kl -mno-sgx -mno-widekl -mshstk --param=l1-cache-line-size=64 --param=l1-cache-size=32 --param=l2-cache-size=36864 -fPIE -pie -ggdb3 -fvisibility=hidden -Wsign-compare -Wall -Wno-uninitialized -Wignored-qualifiers -Wcast-qual
/usr/bin/x86_64-pc-linux-gnu-g++::: -mtune=core2 -pipe -O3 -flto -frecord-gcc-switches -Wstringop-overread -Wformat -Wformat-security -D_FORTIFY_SOURCE=3 -D_GLIBCXX_ASSERTIONS -ftrivial-auto-var-init=pattern -Wl,-z,relro,-z,now -fstack-protector-strong -fstack-clash-protection -fcf-protection=full -Werror=strict-aliasing  -Werror=lto-type-mismatch -Werror=stringop-overread -Werror=format-security -march=alderlake -mabm -mno-cldemote -mno-kl -mno-sgx -mno-widekl -mshstk --param=l1-cache-line-size=64 --param=l1-cache-size=32 --param=l2-cache-size=36864 -fPIE -pie -ggdb3 -fPIC -fPIC -fPIC -fPIC -fvisibility=hidden -Wsign-compare -Wall -Wno-uninitialized -Wignored-qualifiers -Wcast-qual
-- NO PERF COUNTERS
-- User set python executable /usr/bin/python3.12
CMake Warning (dev) at cmake/Modules/GrPython.cmake:21 (find_package):
  Policy CMP0148 is not set: The FindPythonInterp and FindPythonLibs modules
  are removed.  Run "cmake --help-policy CMP0148" for policy details.  Use
  the cmake_policy command to set the policy and suppress this warning.

Call Stack (most recent call first):
  CMakeLists.txt:246 (include)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found PythonInterp: /usr/bin/python3.12 (found suitable version "3.12.8", minimum required is "3.7.2")
CMake Warning (dev) at cmake/Modules/GrPython.cmake:27 (find_package):
  Policy CMP0148 is not set: The FindPythonInterp and FindPythonLibs modules
  are removed.  Run "cmake --help-policy CMP0148" for policy details.  Use
  the cmake_policy command to set the policy and suppress this warning.

Call Stack (most recent call first):
  CMakeLists.txt:246 (include)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found PythonLibs: /usr/lib64/libpython3.12.so (found suitable exact version "3.12.8")
-- Python checking for packaging - found
-- Python checking for numpy - found
-- Python checking for pygccxml - found
CMake Warning (dev) at /usr/share/cmake/pybind11/FindPythonLibsNew.cmake:101 (message):
  Policy CMP0148 is not set: The FindPythonInterp and FindPythonLibs modules
  are removed.  Run "cmake --help-policy CMP0148" for policy details.  Use
  the cmake_policy command to set the policy and suppress this warning, or
  preferably upgrade to using FindPython, either by calling it explicitly
  before pybind11, or by setting PYBIND11_FINDPYTHON ON before pybind11.
Call Stack (most recent call first):
  /usr/share/cmake/pybind11/pybind11Tools.cmake:50 (find_package)
  /usr/share/cmake/pybind11/pybind11Common.cmake:228 (include)
  /usr/share/cmake/pybind11/pybind11Config.cmake:250 (include)
  CMakeLists.txt:276 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found PythonInterp: /usr/bin/python3.12 (found suitable version "3.12.8", minimum required is "3.7")
-- Found PythonLibs: /usr/lib64/libpython3.12.so
-- Performing Test HAS_FLTO
-- Performing Test HAS_FLTO - Success
-- Found pybind11: /usr/include (found version "2.13.6")
-- 
-- Configuring python-support support...
--   Dependency PYTHONLIBS_FOUND = TRUE
--   Dependency PACKAGING_FOUND = TRUE
--   Dependency pybind11_FOUND = 1
--   Dependency NUMPY_FOUND = TRUE
--   Enabling python-support support.
--   Override with -DENABLE_PYTHON=ON/OFF
-- 
-- Configuring VOLK support...
-- Found Volk: Volk::volk
--   Found VOLK:
--   * Version: 3.1.2
--   * Libraries: Volk::volk
--   * Includes: /usr/include
-- Not using additional GNU Radio native architecture optimizations.
-- Performing Test HAVE_CX_LIMITED_RANGE
-- Performing Test HAVE_CX_LIMITED_RANGE - Success
-- Found PkgConfig: /usr/bin/x86_64-pc-linux-gnu-pkg-config (found version "2.3.0")
-- Checking for module 'gmp'
--   Found gmp, version 6.3.0
-- Found GMP: /usr/lib64/libgmpxx.so
-- Using GMP.
-- Found MPLIB: /usr/lib64/libgmpxx.so
-- 
-- Configuring post-install support...
--   Disabling post-install support.
--   Override with -DENABLE_POSTINSTALL=ON/OFF
-- Found Doxygen: /usr/bin/doxygen (found version "1.12.0") found components: doxygen dot
CMake Warning (dev) at docs/CMakeLists.txt:12 (find_package):
  Policy CMP0144 is not set: find_package uses upper-case <PACKAGENAME>_ROOT
  variables.  Run "cmake --help-policy CMP0144" for policy details.  Use the
  cmake_policy command to set the policy and suppress this warning.

  CMake variable MATHJAX2_ROOT is set to:

    /usr/share/mathjax

  For compatibility, find_package is ignoring the variable, but code in a
  .cmake module might still use it.
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Could NOT find MathJax2 (missing: MATHJAX2_JS_PATH) 
-- 
-- Configuring doxygen support...
--   Dependency DOXYGEN_FOUND = YES
--   Disabling doxygen support.
--   Override with -DENABLE_DOXYGEN=ON/OFF
-- 
-- Configuring man-pages support...
--   Disabling man-pages support.
--   Override with -DENABLE_MANPAGES=ON/OFF
-- Using GMP.
-- Looking for C++ include sys/types.h
-- Looking for C++ include sys/types.h - found
-- Looking for C++ include sys/mman.h
-- Looking for C++ include sys/mman.h - found
-- Looking for C++ include sys/ipc.h
-- Looking for C++ include sys/ipc.h - found
-- Looking for C++ include sys/shm.h
-- Looking for C++ include sys/shm.h - found
-- Looking for C++ include signal.h
-- Looking for C++ include signal.h - found
-- Performing Test HAVE_SIGACTION
-- Performing Test HAVE_SIGACTION - Success
-- Performing Test HAVE_SYSCONF
-- Performing Test HAVE_SYSCONF - Success
-- Performing Test HAVE_GETPAGESIZE
-- Performing Test HAVE_GETPAGESIZE - Success
-- Performing Test HAVE_PTHREAD_SIGMASK
-- Performing Test HAVE_PTHREAD_SIGMASK - Success
-- Performing Test HAVE_CREATEFILEMAPPING
-- Performing Test HAVE_CREATEFILEMAPPING - Failed
-- Performing Test HAVE_SHM_OPEN
-- Performing Test HAVE_SHM_OPEN - Success
-- Performing Test HAVE_SINCOS
-- Performing Test HAVE_SINCOS - Success
-- Performing Test HAVE_SINCOSF
-- Performing Test HAVE_SINCOSF - Success
-- Performing Test HAVE_SINF
-- Performing Test HAVE_SINF - Success
-- Performing Test HAVE_COSF
-- Performing Test HAVE_COSF - Success
-- Performing Test HAVE_MMAP
-- Performing Test HAVE_MMAP - Success
-- Performing Test HAVE_PTHREAD_SETSCHEDPARAM
-- Performing Test HAVE_PTHREAD_SETSCHEDPARAM - Success
-- Performing Test HAVE_SCHED_SETSCHEDULER
-- Performing Test HAVE_SCHED_SETSCHEDULER - Success
-- Found Threads: TRUE
-- 
-- Configuring gnuradio-runtime support...
--   Dependency Boost_FOUND = TRUE
--   Dependency PYTHONINTERP_FOUND = TRUE
--   Dependency MPLIB_FOUND = TRUE
--   Dependency spdlog_FOUND = 1
--   Dependency Volk_FOUND = 1
--   Enabling gnuradio-runtime support.
--   Override with -DENABLE_GNURADIO_RUNTIME=ON/OFF
-- 
-- Configuring common-precompiled-headers support...
--   Dependency ENABLE_GNURADIO_RUNTIME = ON
--   Dependency GR_CMAKE_HAS_PCH = 3.30.6
--   Disabling common-precompiled-headers support.
--   Override with -DENABLE_COMMON_PCH=ON/OFF
-- 
-- Configuring gr-ctrlport support...
--   Dependency Boost_FOUND = TRUE
--   Dependency ENABLE_GNURADIO_RUNTIME = ON
--   Disabling gr-ctrlport support.
--   Override with -DENABLE_GR_CTRLPORT=ON/OFF
-- Checking for module 'thrift'
--   Package 'thrift' not found
-- thrift looking for version 0.13
-- Binary 'thrift' not found.
-- Looking for C++ include sys/resource.h
-- Looking for C++ include sys/resource.h - found
-- Checking for module 'libunwind'
--   Found libunwind, version 1.8.1
-- Found libunwind 
-- Loading build date Tue, 17 Dec 2024 21:00:04Z into constants...
-- Loading version v3.10.11.0-1-gee27d6f3 into constants...
-- TRY_SHM_VMCIRCBUF set to ON.
-- 
-- Python checking for PyYAML >= 3.11 - found
-- Python checking for mako >= 1.1.0 - found
-- Python checking for pygobject >= 2.28.6 - found
-- Python checking for Gtk (GI) >= 3.10.8 - found
-- Python checking for Cairo (GI) >= 1.0 - found
-- Python checking for PangoCairo (GI) >= 1.0 - found
-- Python checking for numpy - found
-- Python checking for jsonschema - found
-- 
-- Configuring gnuradio-companion support...
--   Dependency ENABLE_GNURADIO_RUNTIME = ON
--   Dependency ENABLE_PYTHON = ON
--   Dependency PYYAML_FOUND = TRUE
--   Dependency MAKO_FOUND = TRUE
--   Dependency PYGI_FOUND = TRUE
--   Dependency GTK_GI_FOUND = TRUE
--   Dependency CAIRO_GI_FOUND = TRUE
--   Dependency PANGOCAIRO_GI_FOUND = TRUE
--   Dependency NUMPY_FOUND = TRUE
--   Enabling gnuradio-companion support.
--   Override with -DENABLE_GRC=ON/OFF
-- 
-- Configuring JSON/YAML config blocks support...
--   Dependency ENABLE_GRC = ON
--   Dependency JSONSCHEMA_FOUND = TRUE
--   Disabling JSON/YAML config blocks support.
--   Override with -DENABLE_JSONYAML_BLOCKS=ON/OFF
-- Python checking for pytest - found
-- Checking for module 'sndfile'
--   Found sndfile, version 1.2.2
-- Found SNDFILE: /usr/lib64/libsndfile.so
-- 
-- Configuring gr-blocks support...
--   Dependency Boost_FOUND = TRUE
--   Dependency ENABLE_GNURADIO_RUNTIME = ON
--   Enabling gr-blocks support.
--   Override with -DENABLE_GR_BLOCKS=ON/OFF
-- Looking for C++ include io.h
-- Looking for C++ include io.h - not found
-- Blocks/tests: adding benchmark_nco
-- Blocks/tests: adding benchmark_vco
-- Found GSL: /usr/include (found version "2.7.1")
-- 
-- Configuring gr-fec support...
--   Dependency ENABLE_GNURADIO_RUNTIME = ON
--   Dependency ENABLE_GR_BLOCKS = ON
--   Enabling gr-fec support.
--   Override with -DENABLE_GR_FEC=ON/OFF
-- Checking for module 'fftw3f >= 3.0'
--   Found fftw3f, version 3.3.10
-- Found FFTW3f: /usr/lib64/libfftw3f.so
-- 
-- Configuring gr-fft support...
--   Dependency Boost_FOUND = TRUE
--   Dependency ENABLE_GNURADIO_RUNTIME = ON
--   Dependency ENABLE_GR_BLOCKS = ON
--   Dependency FFTW3f_FOUND = TRUE
--   Enabling gr-fft support.
--   Override with -DENABLE_GR_FFT=ON/OFF
-- 
-- Configuring gr-filter support...
--   Dependency Boost_FOUND = TRUE
--   Dependency ENABLE_GNURADIO_RUNTIME = ON
--   Dependency ENABLE_GR_FFT = ON
--   Dependency ENABLE_GR_BLOCKS = ON
--   Enabling gr-filter support.
--   Override with -DENABLE_GR_FILTER=ON/OFF
-- Python checking for pyqtgraph - found
-- Python checking for scipy - found
-- 
-- Configuring gr-analog support...
--   Dependency ENABLE_GNURADIO_RUNTIME = ON
--   Dependency ENABLE_GR_BLOCKS = ON
--   Dependency ENABLE_GR_FFT = ON
--   Dependency ENABLE_GR_FILTER = ON
--   Enabling gr-analog support.
--   Override with -DENABLE_GR_ANALOG=ON/OFF
-- 
-- Configuring gr-digital support...
--   Dependency Boost_FOUND = TRUE
--   Dependency ENABLE_GNURADIO_RUNTIME = ON
--   Dependency ENABLE_GR_FFT = ON
--   Dependency ENABLE_GR_FILTER = ON
--   Dependency ENABLE_GR_BLOCKS = ON
--   Dependency ENABLE_GR_ANALOG = ON
--   Enabling gr-digital support.
--   Override with -DENABLE_GR_DIGITAL=ON/OFF
-- 
-- Configuring gr-dtv support...
--   Dependency ENABLE_GNURADIO_RUNTIME = ON
--   Dependency ENABLE_GR_ANALOG = ON
--   Dependency ENABLE_GR_FILTER = ON
--   Dependency ENABLE_GR_FEC = ON
--   Enabling gr-dtv support.
--   Override with -DENABLE_GR_DTV=ON/OFF
-- Performing Test SSE2_SUPPORTED
-- Performing Test SSE2_SUPPORTED - Success
-- 
-- Configuring gr-audio support...
--   Dependency ENABLE_GNURADIO_RUNTIME = ON
--   Enabling gr-audio support.
--   Override with -DENABLE_GR_AUDIO=ON/OFF
-- Checking for module 'alsa'
--   Found alsa, version 1.2.12
-- Found ALSA 1.2.12
-- Checking for module 'jack'
--   Package 'jack' not found
-- Looking for C++ include AudioUnit/AudioUnit.h
-- Looking for C++ include AudioUnit/AudioUnit.h - not found
-- Looking for C++ include AudioToolbox/AudioToolbox.h
-- Looking for C++ include AudioToolbox/AudioToolbox.h - not found
-- Checking for module 'portaudio-2.0'
--   Found portaudio-2.0, version 19
-- 
-- Configuring gr-channels support...
--   Dependency ENABLE_GNURADIO_RUNTIME = ON
--   Dependency ENABLE_GR_BLOCKS = ON
--   Dependency ENABLE_GR_FFT = ON
--   Dependency ENABLE_GR_FILTER = ON
--   Dependency ENABLE_GR_ANALOG = ON
--   Enabling gr-channels support.
--   Override with -DENABLE_GR_CHANNELS=ON/OFF
-- 
-- Configuring gr-pdu support...
--   Dependency ENABLE_GNURADIO_RUNTIME = ON
--   Enabling gr-pdu support.
--   Override with -DENABLE_GR_PDU=ON/OFF
-- Found libiio: /usr/lib64/libiio.so (found version "0.25")
-- Found libad9361: /usr/lib64/libad9361.so (found version "0.2")
-- 
-- Configuring gr-iio support...
--   Dependency ENABLE_GNURADIO_RUNTIME = ON
--   Dependency ENABLE_GR_BLOCKS = ON
--   Dependency libiio_FOUND = TRUE
--   Enabling gr-iio support.
--   Override with -DENABLE_GR_IIO=ON/OFF
-- Performing Test libad9361_SUFFICIENT
-- Performing Test libad9361_SUFFICIENT - Success
-- Python checking for PyQt5 - found
-- Checking for module 'Qt5Qwt6'
--   Found Qt5Qwt6, version 6.2.0
-- QWT Version: 6.2.0
-- Found Qwt: /usr/lib64/libqwt6-qt5.so
-- 
-- Configuring gr-qtgui support...
--   Dependency QT_FOUND = 1
--   Dependency QWT_FOUND = TRUE
--   Dependency ENABLE_GNURADIO_RUNTIME = ON
--   Dependency ENABLE_GR_FFT = ON
--   Dependency ENABLE_GR_FILTER = ON
--   Dependency PYTHONLIBS_FOUND = TRUE
--   Dependency PYQT5_FOUND = TRUE
--   Enabling gr-qtgui support.
--   Override with -DENABLE_GR_QTGUI=ON/OFF
-- 
-- Configuring gr-trellis support...
--   Dependency ENABLE_GNURADIO_RUNTIME = ON
--   Dependency ENABLE_GR_ANALOG = ON
--   Dependency ENABLE_GR_BLOCKS = ON
--   Dependency ENABLE_GR_DIGITAL = ON
--   Enabling gr-trellis support.
--   Override with -DENABLE_GR_TRELLIS=ON/OFF
-- Found UHD: /usr/lib64/libuhd.so (Required is at least version "3.9.7")
-- Python checking for PyQt5 - found
-- 
-- Configuring gr-uhd support...
--   Dependency Boost_FOUND = TRUE
--   Dependency UHD_FOUND = TRUE
--   Dependency ENABLE_GNURADIO_RUNTIME = ON
--   Dependency ENABLE_GR_FILTER = ON
--   Dependency ENABLE_GR_BLOCKS = ON
--   Dependency ENABLE_GR_ANALOG = ON
--   Enabling gr-uhd support.
--   Override with -DENABLE_GR_UHD=ON/OFF
-- 
-- Configuring gr-uhd UHD 4.0 RFNoC support...
--   Dependency ENABLE_GR_UHD = ON
--   Dependency UHD_FOUR_POINT_OH_RFNOC = TRUE
--   Disabling gr-uhd UHD 4.0 RFNoC support.
--   Override with -DENABLE_UHD_RFNOC=ON/OFF
--   UHD Version: 4.6.0.0
-- Python checking for Mako >= 1.1.0 - found
-- Python checking for click - found
-- 
-- Configuring gr-utils support...
--   Dependency ENABLE_GNURADIO_RUNTIME = ON
--   Dependency ENABLE_PYTHON = ON
--   Dependency MAKO_FOUND = TRUE
--   Enabling gr-utils support.
--   Override with -DENABLE_GR_UTILS=ON/OFF
-- 
-- Configuring gr_modtool support...
--   Dependency ENABLE_GNURADIO_RUNTIME = ON
--   Dependency ENABLE_PYTHON = ON
--   Dependency CLICK_FOUND = TRUE
--   Enabling gr_modtool support.
--   Override with -DENABLE_GR_MODTOOL=ON/OFF
-- 
-- Configuring gr_blocktool support...
--   Dependency ENABLE_GNURADIO_RUNTIME = ON
--   Dependency ENABLE_PYTHON = ON
--   Enabling gr_blocktool support.
--   Override with -DENABLE_GR_BLOCKTOOL=ON/OFF
-- Found SDL: /usr/lib64/libSDLmain.a;/usr/lib64/libSDL.so (found version "1.2.68")
-- 
-- Configuring gr-video-sdl support...
--   Dependency SDL_FOUND = TRUE
--   Dependency ENABLE_GNURADIO_RUNTIME = ON
--   Enabling gr-video-sdl support.
--   Override with -DENABLE_GR_VIDEO_SDL=ON/OFF
-- 
-- Configuring gr-vocoder support...
--   Dependency ENABLE_GNURADIO_RUNTIME = ON
--   Dependency ENABLE_GR_FFT = ON
--   Dependency ENABLE_GR_BLOCKS = ON
--   Dependency ENABLE_GR_FILTER = ON
--   Dependency ENABLE_GR_ANALOG = ON
--   Enabling gr-vocoder support.
--   Override with -DENABLE_GR_VOCODER=ON/OFF
-- Found Codec2: /usr/lib64/libcodec2.so
-- Found GSM: /usr/lib64/libgsm.so
-- 
-- Configuring gr-wavelet support...
--   Dependency ENABLE_GNURADIO_RUNTIME = ON
--   Dependency ENABLE_GR_BLOCKS = ON
--   Dependency ENABLE_GR_ANALOG = ON
--   Dependency GSL_FOUND = TRUE
--   Enabling gr-wavelet support.
--   Override with -DENABLE_GR_WAVELET=ON/OFF
-- Checking for module 'libzmq'
--   Found libzmq, version 4.3.5
-- Found ZeroMQ: /usr/lib64/libzmq.so
-- Performing Test CPPZMQ_HAS_SHUTDOWN
-- Performing Test CPPZMQ_HAS_SHUTDOWN - Success
-- 
-- Configuring gr-zeromq support...
--   Dependency ENABLE_GNURADIO_RUNTIME = ON
--   Dependency ZEROMQ_FOUND = TRUE
--   Dependency CPPZMQ_HAS_SHUTDOWN = 1
--   Enabling gr-zeromq support.
--   Override with -DENABLE_GR_ZEROMQ=ON/OFF
-- 
-- Configuring gr-network support...
--   Dependency Boost_FOUND = TRUE
--   Dependency ENABLE_GNURADIO_RUNTIME = ON
--   Enabling gr-network support.
--   Override with -DENABLE_GR_NETWORK=ON/OFF
-- Looking for C++ include windows.h
-- Looking for C++ include windows.h - not found
-- 
-- Configuring gr-soapy support...
--   Dependency SoapySDR_FOUND = 1
--   Dependency ENABLE_GNURADIO_RUNTIME = ON
--   Enabling gr-soapy support.
--   Override with -DENABLE_GR_SOAPY=ON/OFF
--   SOAPY Version: 0.8.1-unknown
-- No C++ unit tests... skipping
-- 
-- ######################################################
-- # Gnuradio enabled components                         
-- ######################################################
--   * testing-support
--   * python-support
--   * gnuradio-runtime
--   * gnuradio-companion
--   * gr-blocks
--   * gr-fec
--   * gr-fft
--   * gr-filter
--   * gr-analog
--   * gr-digital
--   * gr-dtv
--   * gr-audio
--   * * alsa
--   * * oss
--   * gr-channels
--   * gr-pdu
--   * gr-iio
--   * * libad9361
--   * gr-qtgui
--   * gr-trellis
--   * gr-uhd
--   * gr-utils
--   * gr_modtool
--   * gr_blocktool
--   * gr-video-sdl
--   * gr-vocoder
--   * * codec2
--   * * freedv
--   * * gsm
--   * gr-wavelet
--   * gr-zeromq
--   * gr-network
--   * gr-soapy
-- 
-- ######################################################
-- # Gnuradio disabled components                        
-- ######################################################
--   * post-install
--   * doxygen
--   * man-pages
--   * common-precompiled-headers
--   * gr-ctrlport
--   * JSON/YAML config blocks
--   * gr-uhd UHD 4.0 RFNoC
-- 
-- Using install prefix: /usr
-- Building for version: v3.10.11.0-1-gee27d6f3 / 3.10.12git
-- <<< Gentoo configuration >>>
Build type      None
Install path    /usr
Compiler flags:
C               -mtune=core2 -pipe -O3 -flto -frecord-gcc-switches -Wstringop-overread -Wformat -Wformat-security -D_FORTIFY_SOURCE=3 -D_GLIBCXX_ASSERTIONS -ftrivial-auto-var-init=pattern -Wl,-z,relro,-z,now -fstack-protector-strong -fstack-clash-protection -fcf-protection=full -Werror=strict-aliasing -Werror=odr -Werror=lto-type-mismatch -Werror=stringop-overread -Werror=format-security -march=alderlake -mabm -mno-cldemote -mno-kl -mno-sgx -mno-widekl -mshstk --param=l1-cache-line-size=64 --param=l1-cache-size=32 --param=l2-cache-size=36864 -fPIE -pie -ggdb3 -fvisibility=hidden -Wsign-compare -Wall -Wno-uninitialized -Wignored-qualifiers -Wcast-qual
C++             -mtune=core2 -pipe -O3 -flto -frecord-gcc-switches -Wstringop-overread -Wformat -Wformat-security -D_FORTIFY_SOURCE=3 -D_GLIBCXX_ASSERTIONS -ftrivial-auto-var-init=pattern -Wl,-z,relro,-z,now -fstack-protector-strong -fstack-clash-protection -fcf-protection=full -Werror=strict-aliasing  -Werror=lto-type-mismatch -Werror=stringop-overread -Werror=format-security -march=alderlake -mabm -mno-cldemote -mno-kl -mno-sgx -mno-widekl -mshstk --param=l1-cache-line-size=64 --param=l1-cache-size=32 --param=l2-cache-size=36864 -fPIE -pie -ggdb3 -fPIC -fPIC -fPIC -fPIC -fvisibility=hidden -Wsign-compare -Wall -Wno-uninitialized -Wignored-qualifiers -Wcast-qual -fcx-limited-range
Linker flags:
Executable      -Wl,-O1 -Wl,--as-needed -Wl,--defsym=__gentoo_check_ldflags__=0
Module          -Wl,-O1 -Wl,--as-needed -Wl,--defsym=__gentoo_check_ldflags__=0
Shared          -Wl,-O1 -Wl,--as-needed -Wl,--defsym=__gentoo_check_ldflags__=0

-- Configuring done (19.7s)
-- Generating done (0.4s)
```
```

**Comment by ZeroChaos-** [comment]
This is running in Gentoo's installed python 3.12
```
zero@naga gnuradio % python --version                                                                                                                                                (git)-[master]  1 ⨯
Python 3.12.8
```

**Comment by marcusmueller** [comment]
I introduced that bug, but Argilo already fixed it on main in 7d61746

**Comment by marcusmueller** [comment]
Looks like we just need to finish backporting of #7499

**Comment by marcusmueller** [comment]
closed via #7570 . For some reason, the magic close verb on that PR didn't close this automagically.
